system_baseline/openapi/internal_api.spec.yaml

```diff
@@ -110,3 +166,11 @@ components:
       schema:
         $ref: "#/components/schemas/SystemId"
       description: system id to search by
+    SystemIds:
+      in: path
+      name: system_ids
+      required: true
+      schema:
+        type: array
+        items:
+          type: string
```

Comment:
format: uuid here also like above on line 160

Reply:
you are right